lago/dom_template.xml

```diff
@@ -42,6 +42,11 @@
       <console type='pty'>
         <target type='serial' port='0'/>
       </console>
+      <video>
+        <model type='cirrus' vram='16384' heads='1'/>
```

Comment:
Cirrus is a bad choice - we've just switched away from it in oVirt.
Should be VGA.